.circleci/common.yml

```diff
@@ -431,6 +430,10 @@ jobs:
 
             aws lambda wait function-updated --function-name '<< parameters.function-name >>'
 
+            NEW_ENVVARS=$(aws lambda get-function-configuration --function-name '<< parameters.function-name >>' --query "Environment.Variables | merge(@, \`{\"GIT_SHA\":\"$CIRCLE_SHA1\"}\`)")
```

Comment:
AWS doesn't have a way to update 1 environment variable on a function, so we have to get the existing ones, then replace it with the one we want.